.ci/baseline.benchmark

```diff
@@ -0,0 +1,15 @@
+Running benchmark 1/3...
+================== BENCHMARK RESULT #1 ==================
+Execution Time: 8.82 s
+============================================================
+Running benchmark 2/3...
+================== BENCHMARK RESULT #2 ==================
+Execution Time: 8.37 s
+============================================================
+Running benchmark 3/3...
+================== BENCHMARK RESULT #3 ==================
+Execution Time: 8.50 s
+============================================================
+===================== BENCHMARK SUMMARY ====================
+Average Execution Time: 8.56 s
```

Comment:
This baseline file is generated by the shell script and is parsable by the same shell script. So whenever we want to change the baseline, we can use the same tool to generate this.